docs/error-reporting/platform-integrations/source-map.md

```diff
@@ -9,88 +9,259 @@ import Tabs from '@theme/Tabs';
 
-Use the following command to install backtrace-sourcemap-tools as a dev dependency:
+## General Development
+
+### Step 1: Enable Source Maps for Your Application
+
+This step is development tool specific.
+
+#### Typescript (tsc)
+
+Set `sourceMap` in `compilerOptions` in your `tsconfig.json` to `true`. For example:
+```jsonc
+{
+    "compilerOptions": {
+        // other options
+        "sourceMap": true
+    },
+    "include": ["./src"]
+}
+```
+
+#### UglifyJS
+
+Pass `--source-map` as parameter to `uglifyjs`:
+
+```
+$ uglifyjs main.js -c -m --source-map -o main.min.js
+```
+
+### Step 2: Set up `@backtrace/javascript-cli`
+
+1. Install `@backtrace/javascript-cli` as a dev dependency:
 
 ```bash
-npm install "git@github.com:backtrace-labs/backtrace-sourcemap-tools.git" --save-dev
+> npm install --save-dev @backtrace/javascript-cli
 ```
 
-Alternatively, add the following entry to your package.json file under `devDependencies`:
+2. Add the following scripts to your `package.json` file:
 
 ```json
-"devDependencies": {
-  "backtrace-sourcemap-tools": "git@github.com:backtrace-labs/backtrace-sourcemap-tools.git"
+"scripts": {
+  "backtrace:process": "backtrace-js process OUTPUT_DIRECTORY",
+  "backtrace:upload": "backtrace-js upload OUTPUT_DIRECTORY UPLOAD_OPTIONS"
 }
 ```
 
-## Step 3: Configure Sourcemap Upload URL in `package.json`
-
-Specify the URL, token, and universe where Backtrace sourcemap tools will upload sourcemap files when your project is built. Add the following configuration to your `package.json` file under the top-level key `backtrace`:
+You can also use `npx` and skip adding the dependency:
 
 ```json
-"backtrace": {
-  "sourcemap": {
-    "upload": "https://<universe>.sp.backtrace.io:6098/post?format=sourcemap&token=<token>"
-  }
+"scripts": {
+  "backtrace:process": "npx --package @backtrace/javascript-cli backtrace-js process OUTPUT_DIRECTORY",
+  "backtrace:upload": "npx --package @backtrace/javascript-cli backtrace-js upload OUTPUT_DIRECTORY UPLOAD_OPTIONS"
 }
 ```
 
-You need to replace `<universe>` with your universe name, and `<token>` with the symbol upload token generated in your Project Settings under the Symbols section. For on-premise Backtrace servers, replace `<universe>.sp.backtrace.io:6098` with the server and port of your Backtrace instance.
+Ensure to replace `OUTPUT_DIRECTORY` with the path to the directory where your transpiled scripts are stored.
 
-## Step 4: Set Up Automatic Sourcemap Uploads
+Replace `UPLOAD_OPTIONS` with either `--subdomain <your subdomain> --token <your token>` or `--url <your upload URL>`, obtained from [the symbol submission URL](#obtaining-symbol-submission-url).
+
+#### Configuration File
+Instead of providing options in script argument lines, you can configure them in the `.backtraceclirc` configuration file:
+
+```jsonc
+{
+  "path": "OUTPUT_DIRECTORY",
+  "upload": {
+    "subdomain": "your subdomain",
+    "token": "your token",
+    // or
```

Comment:
Can we remove this option, and only present the submit url? Less choices == more better.

Reply:
Resolved with your PR.